tests/path_gateway_raw_test.go

```diff
@@ -157,3 +143,115 @@ func TestGatewayBlock(t *testing.T) {
 
 	RunWithSpecs(t, tests, specs.PathGatewayRaw)
 }
+
+func TestGatewayBlockMultiRangeRequest(t *testing.T) {
+	// Multi-range requests MUST conform to the HTTP semantics. The server does not
+	// need to be able to support returning multiple ranges. However, it must respond
+	// correctly.
+	fixture := car.MustOpenUnixfsCar("gateway-raw-block.car")
+
+	var (
+		contentType  string
+		contentRange string
+	)
+
+	RunWithSpecs(t, SugarTests{
+		{
+			Name: "GET with application/vnd.ipld.raw with multiple range request includes correct bytes",
+			Request: Request().
+				Path("/ipfs/{{cid}}/dir/ascii.txt", fixture.MustGetCid()).
+				Headers(
+					Header("Accept", "application/vnd.ipld.raw"),
+					Header("Range", "bytes=6-16,0-4"),
+				),
+			Response: Expect().
+				Status(206).
+				Headers(
+					Header("Content-Type").
+						Checks(func(v string) bool {
+							contentType = v
+							return v != ""
+						}),
+					Header("Content-Range").
+						Checks(func(v string) bool {
+							contentRange = v
+							return true
+						}),
```

Comment:
@laurentsenta these are mostly for collecting the headers. `Content-Type` needs to be present, but `Content-Range` might not be present.